src/basics.md

```diff
@@ -1092,6 +1093,35 @@ fn main() {
 }
 ```
 
+[ex-format-datetime]: #ex-format-datetime
+<a name="ex-format-datetime"></a>
+## Display formatted date and time
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Displays UTC in the well-known formats RFC 2822 and RFC 3339, and in a custom
+format.
+
+```rust
+extern crate chrono;
+use chrono::{DateTime, Utc};
+
+fn main() {
+    // Get coordinated universal time (UTC)
```

Comment:
I'd move all the comments to the textual description above with key identifiers hyperlinked to the chrono docs.rs (please remember to link to wildcard "*" version)